doc/help/kdb-spec-mount.md

```diff
@@ -92,10 +92,10 @@ the `spec` plugin is present:
 
 ## EXAMPLES
 
-To mount /example as described in `spec/example`:  
```

Comment:
As far as I can tell, we only need the whitespace if we want the Markdown text rendered like this:

> To mount /example as described in `spec/example`:  
> `kdb spec-mount /example`

What’s the problem with the text rendering:

> To mount /example as described in `spec/example`: `kdb spec-mount /example`

? If you want a manual line break, then how about 

> To mount /example as described in `spec/example`:
>
> `kdb spec-mount /example`

?